src/memory_targets/memory_target_numa.c

```diff
@@ -148,13 +148,11 @@ static umf_result_t numa_get_capacity(void *memTarget, size_t *capacity) {
     return UMF_RESULT_SUCCESS;
 }
 
-static umf_result_t numa_get_bandwidth(void *srcMemoryTarget,
-                                       void *dstMemoryTarget,
-                                       size_t *bandwidth) {
-    if (!srcMemoryTarget || !dstMemoryTarget || !bandwidth) {
-        return UMF_RESULT_ERROR_INVALID_ARGUMENT;
-    }
+enum memattr_type { MEMATTR_TYPE_BANDWIDTH, MEMATTR_TYPE_LATENCY };
```

Comment:
please add typedef so this matches the coding style in our project

Reply:
Done.